Export VmRepl

Sorry @NicoNex I was wrong 😅

```#!sh
(⎈ |local:default)
prologic@Jamess-iMac
Wed Aug 03 02:09:08
~/Projects/gonix
 (tau_applet) 0
$ make
# git.mills.io/prologic/gonix/internal/applets/tau
internal/applets/tau/tau.go:67:11: undefined: tau.VmREPL
make: *** [rootfs/sbin/coreutils] Error 2
```

I don't want to re-invent the REPL. 😂

--- a/cmd/tau/main.go
+++ b/cmd/tau/main.go
@@ -26,9 +26,9 @@ func main() {
 
 	default:
 		if useEval {
-			evalREPL()
+			tau.EvalREPL()
 		} else {
-			vmREPL()
+			tau.VmREPL()
 		}
 	}
 }

--- a/repl.go
+++ b/repl.go
@@ -1,7 +1,7 @@
 //go:build !windows
 // +build !windows
 
-package main
+package tau
 
 import (
 	"fmt"
@@ -17,7 +17,7 @@ import (
 	"github.com/NicoNex/tau/internal/vm"
 )
 
-func evalREPL() error {
+func EvalREPL() error {
 	var env = obj.NewEnv()
 
 	initState, err := term.MakeRaw(0)
@@ -54,7 +54,7 @@ func evalREPL() error {
 	}
 }
 
-func vmREPL() error {
+func VmREPL() error {
 	var (
 		consts      []obj.Object
 		globals     = make([]obj.Object, vm.GlobalSize)

--- a/repl_win.go
+++ b/repl_win.go
@@ -1,7 +1,7 @@
 //go:build windows
 // +build windows
 
-package main
+package tau
 
 import (
 	"bufio"
@@ -15,7 +15,7 @@ import (
 	"github.com/NicoNex/tau/internal/vm"
 )
 
-func evalREPL() {
+func EvalREPL() {
 	var (
 		env    = obj.NewEnv()
 		reader = bufio.NewReader(os.Stdin)
@@ -46,7 +46,7 @@ func evalREPL() {
 	}
 }
 
-func vmREPL() {
+func VmREPL() {
 	var (
 		consts      []obj.Object
 		globals     = make([]obj.Object, vm.GlobalSize)

